examples/printerdemo/ui/printer_queue.slint

```diff
@@ -159,8 +159,8 @@ component NarrowPrintQueueElement inherits Rectangle {
                 Text {
                     // TODO: text-transform: uppercase
                     text: {
-                        if (root.queue-item.status == "PRINTING") {
-                        "\{root.queue-item.progress}% - \{root.queue-item.status}"
+                        if (root.queue-item.progress > 0) {
+                            @tr("{}% - {}",root.queue-item.progress,  root.queue-item.status)
```

Comment:
This calls for context :)

Reply:
Either context, or named argument

Somehing like one of:

```
 @tr("{progress}% - {status}",  progress: root.queue-item.progress,  status: root.queue-item.status);
 @tr("{progress}% - {status}",  progress = root.queue-item.progress,  status = root.queue-item.status);
```